app.js

```diff
@@ -35,9 +35,9 @@ socket.on('general', (data) => {
     const min = date.getMinutes()
     //TODO: Fix the % sign showing up
     if (min < 10) {
-        console.log(Colors.cyan,`${hour}:0${min} :`, Colors.yellowBgBlackLt,` ${data.name}`)
+        process.stdout.write(Colors.cyan +`${hour}:0${min} : ` + Colors.reset + Colors.yellowBgBlackLt +`${data.name}` +Colors.reset +"\n")
     } else {
-        console.log(Colors.cyan,`${hour}:${min} :`, Colors.yellowBgBlackLt, ` ${data.name}`)
+        process.stdout.write(Colors.cyan +`${hour}:${min} : ` + Colors.reset + Colors.yellowBgBlackLt +`${data.name}` +Colors.reset +"\n")
```

Comment:
Sure, I think it's a good idea since it's already in a string literal. (You mean only in this line right?)

@Tuanhao String literals: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals